system/ee/ExpressionEngine/Addons/channel/libraries/channel_form/Channel_form_session.php

```diff
@@ -52,10 +52,10 @@ public function __construct($config)
     /**
      * userdata
      *
-     * @param	mixed $str
-     * @return	void
+     * @param mixed $str
+     * @return void
      */
-    public function userdata($str)
+    public function userdata($str, $default = false)
```

Comment:
@intoeetive I'm wondering why this $default value was added, but doesn't appear to be used?

Reply:
This is needed because the class
`class Channel_form_session extends EE_Session`
and the function definition needs to be compatible with definition in parent class, where `$default` exists